README.md

```diff
@@ -17,3 +18,48 @@ In order to run the MIMIC-IV ETL, simply run the following command:
 In order to run the MIMIC-IV ETL, simply run the following command:
 
 `meds_etl_omop omop omop-esds` where omop is a folder containing csv files (optionally gzipped) for an OMOP dataset. Each table should either be a csv file with the table name (such as person.csv) or a folder with the table name containing csv files.
+
+## MEDS CSV
```

Comment:
I like @mmcdermott's suggestion of "MEDS Flat" being thought of as an alternate data structure to MEDS. It may sometimes be used as a convenient intermediate in ETLs, but it may have many use cases. Nice to have a consistent flat format that people can use as an input/output to MEDS when needed. 

In this case, I feel like there should be `flatten()` (or `to_meds_flat()`) method on the MEDS object, presumably implemented at https://github.com/Medical-Event-Data-Standard/meds/tree/main and then used here?

Reply:
I added a reverse ETL to this code to support the inverse transformation. I agree we might want to add a flat file schema to the meds repo as well. I wouldn't do flatten or to_flat_file.

I would simply implement `def flat_patient_schema()` in addition to our current `def patient_schema()`.